Sigil/Sigil.nuspec

```diff
@@ -3,15 +3,15 @@
   <metadata>
     <id>sigil</id>
     <title>Sigil</title>
-    <version>0.8.4</version>
+    <version>0.8.7</version>
     <authors>John Nachtimwald</authors>
     <owners>Damien Ryan</owners>
-    <summary>Sigil 0.8.4 ePub Editor.</summary>
+    <summary>Sigil ePub Editor.</summary>
     <description>Sigil is a free, open source, multi-platform ebook editor. It is designed to edit books in ePub format.</description>
-    <projectUrl>https://code.google.com/p/sigil/</projectUrl>
+    <projectUrl>http://sigil-ebook.com/</projectUrl>
     <tags>epub editor books</tags>
     <licenseUrl>https://github.com/user-none/Sigil/blob/master/COPYING.txt</licenseUrl>
-    <iconUrl>https://cdn.rawgit.com/user-none/Sigil/master/src/Sigil/Resource_Files/icon/app_icon_128.png</iconUrl>
+    <iconUrl>https://cdn.rawgit.com/Sigil-Ebook/Sigil/7cca9b68375b7735a16f96c49cc60e34b959b7ea/src/Resource_Files/icon/app_icon_128.png</iconUrl>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
```

Comment:
It is officially recommended in the [packaging guidelines](https://github.com/chocolatey/choco/wiki/CreatePackages#package-icon-guidelines) to use https://cdn.rawgit.com with commit-specific URLs, not branch-specific URLs like master.

This is to avoid exactly that problem that happened with this package. The software author altered the icon path in the repository, causing the package’s iconUrl respond with 404. https://cdn.rawgit.com caches files *permanently*, which avoids these problems completely.